{"name": "test_admin_login[case2_用户名错误的登录]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <case.test_member_controller.test_login.Test_DengLu object at 0x000001B4257D0438>\ncase_data = {'body': {'password': '12345678', 'username': 'adminddd'}, 'case_name': 'case2_用户名错误的登录', 'expect': {'code': 500, 'message': 'Bad credentials'}}\nmember_controller_z = <api.admin.member_controller_api.MemberControllerApi object at 0x000001B4257F1668>\n\n    @decorate_log\n    @pytest.mark.parametrize(\"case_data\", data,ids=[data[i]['case_name'] for i in range(len(data))])  # 参数化测试用例\n    \n    def test_admin_login(self,case_data,member_controller_z):\n        # time.sleep(3)\n        print(\"case_data\",case_data)\n        case_body = case_data['body']\n        \"\"\"请求登录测试用例\"\"\"\n        with allure.step('登录步骤'):\n            actual_data = member_controller_z.login(password=case_body['password']\n                           ,username=case_body['username'])\n            \"\"\"断言\"\"\"\n>           assert_api(actual_data,case_data['expect'])\n\ntest_login.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\common\\logger.py:68: in log\n    raise e\n..\\..\\common\\logger.py:64: in log\n    func(*args, **kwargs)\n..\\..\\common\\assert_api.py:36: in assert_api\n    raise e\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nactual_data = {'code': 200, 'data': {'expiresIn': 86399, 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZ...3mSpGX2MFJbcpoIZTByg27nzAJfkK9J57h_aPZkkMcx0raAPVhTnMl9zUzmI5wsb_3qofCTao', 'tokenHead': 'Bearer '}, 'message': '操作成功'}\nexpect_data = {'code': 500, 'message': 'Bad credentials'}\n\n    @decorate_log\n    def assert_api(actual_data,expect_data):\n        # Logger.info(\"*************** {}开始执行用例 ***************\".format(expect_data['info']))\n        Logger.info(\"\\r\\nassert:\\r\\n 实际结果：{}，\\r\\n期望结果{}\".format(actual_data, expect_data))\n        try:\n            a = expect_data\n            b = actual_data\n            for i, j in a.items():\n                if i in b.keys():\n>                   assert j == b[i]\nE                   AssertionError\n\n..\\..\\common\\assert_api.py:27: AssertionError"}, "steps": [{"name": "登录步骤", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\case\\test_member_controller\\test_login.py\", line 37, in test_admin_login\n    assert_api(actual_data,case_data['expect'])\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 68, in log\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 64, in log\n    func(*args, **kwargs)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 36, in assert_api\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 27, in assert_api\n    assert j == b[i]\n"}, "steps": [{"name": "响应日志", "status": "passed", "parameters": [{"name": "status_code", "value": "200"}, {"name": "text", "value": "{'code': 200, 'message': '操作成功', 'data': {'token': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4MTk0ODI2LCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiMmM2YTI2MWMtZTQ3Ny00ODBlLTk2YjMtODU5ZmYyZmYyNTI2IiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.e8b_A1jSgcvQZo6ohS7decTBZe0km7dvcDD0ZwS2Lb8MM93iiNRPIiAFwGGo0Ekkv0YmOyVQ0bWpielIBvCNJS5HpHFR4rJ8Ln3mSpGX2MFJbcpoIZTByg27nzAJfkK9J57h_aPZkkMcx0raAPVhTnMl9zUzmI5wsb_3qofCTao', 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJhdGkiOiIyYzZhMjYxYy1lNDc3LTQ4MGUtOTZiMy04NTlmZjJmZjI1MjYiLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4NzEzMjI2LCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiZGM2MzExN2ItYTU4ZS00ZWE3LTk3OTUtNDZlNmYyMmVmMzFkIiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.USrEne9SY2oZkI_a5ez3qUmu4ZzAAYzMjQoAo1NB0yTLY4D-5CjUJOWSu5RhfoMdZWCR9cku4yhvjTcqVONtfLk3_ak7kWGBNhkxzNdGaFRLO4658_oUGLe7Ncqm0FRp4Ti6xHkwNj--7-S6R29H72vHGzF-fTuNPwLanIjNrUQ', 'tokenHead': 'Bearer ', 'expiresIn': 86399}}"}], "start": 1648108426311, "stop": 1648108426311}, {"name": "请求日志", "status": "passed", "parameters": [{"name": "url", "value": "'http://121.37.169.128:8201/mall-member/sso/login?platform=test&time=1648108425563&username=admin&password=123456'"}, {"name": "method", "value": "'post'"}, {"name": "body", "value": "{}"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}], "start": 1648108426311, "stop": 1648108426311}], "start": 1648108425563, "stop": 1648108426312}], "attachments": [{"name": "log", "source": "075fbd20-1ed3-4dfe-a98e-0ccd646c603d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_data", "value": "{'case_name': 'case2_用户名错误的登录', 'body': {'username': 'adminddd', 'password': '12345678'}, 'expect': {'message': 'Bad credentials', 'code': 500}}"}], "start": 1648108425562, "stop": 1648108426313, "uuid": "02b0ced5-7b87-49fa-8a4c-3e158339e193", "historyId": "0f326f180f219452afd11079cb76c2e0", "testCaseId": "02c7f242c9f09fbdefa4b8f5b820915e", "fullName": "case.test_member_controller.test_login.Test_DengLu#test_admin_login", "labels": [{"name": "feature", "value": "测试member_controller模块"}, {"name": "story", "value": "登录功能"}, {"name": "parentSuite", "value": "case.test_member_controller"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_DengLu"}, {"name": "host", "value": "LAPTOP-9964GMTD"}, {"name": "thread", "value": "53136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_member_controller.test_login"}]}